pod/perlpacktut.pod

```diff
@@ -58,17 +57,21 @@ corresponding to a byte:
 
 What was in this chunk of memory? Numbers, characters, or a mixture of
 both? Assuming that we're on a computer where ASCII (or some similar)
-encoding is used: hexadecimal values in the range C<0x40> - C<0x5A>
+encoding is used: hexadecimal values in the range C<0x41> - C<0x5A>
 indicate an uppercase letter, and C<0x20> encodes a space. So we might
-assume it is a piece of text, which some are able to read like a tabloid;
-but others will have to get hold of an ASCII table and relive that
-firstgrader feeling. Not caring too much about which way to read this,
+assume it is a piece of ASCII text.  We can verify that by doing
+
+ print unpack( 'A*', $mem), "\n"
+
+ A MAN A PLAN A CANAL PANAMA
```

Comment:
This is no different to:
```
print $mem, "\n";
```
I don't see how it adds anything

Reply:
I think this pod needs more extensive changes, so closing this  PR